ci/Jenkinsfiles/build.groovy

```diff
@@ -527,7 +527,7 @@ pipeline {
           ----------------------------------------"""
           runFunctionalTests('ftests')
         }
-        findText regexp: ".*ERROR.*", fileSet: "ftests/**/log/server.log"
+        findText regexp: ".*ERROR.*", fileSet: "ftests/**/log/server.log", buildResult: 'UNSTABLE'
```

Comment:
Maybe adapt the slack notif accordingly (?) see https://github.com/nuxeo/nuxeo-explorer/blob/master/ci/Jenkinsfiles/build.groovy#L412 (where ERROR in logs are frequent too)

Reply:
This change was more a test to help me to understand what is going on. In the end, this stage should make the test FAILURE, the change has been removed.